distribution/docker/docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM knotx/knotx:latest
```

Comment:
`latest` is not good option here, I would force `2.0.1-SNAPSHOT`. 

Please note: if the `master` branch contains some breaking changes in API, the implemented Handler will contain those changes too. Then when you download `2.0` Docker image (this can be `latest`, then your Knot.x instance will fail).